Src/AutoFixture/RandomRangedNumberGenerator.cs

```diff
@@ -135,13 +133,18 @@ private static long ConvertLimit(object limit)
 
                 case TypeCode.UInt64:
                         return (long)(ulong)limit;
-
-               
             }
 
             throw new ArgumentException("Limit parameter is non-numeric ", "limit");         
-        }      
-       
+        }
+
+        private static TypeCode GetTypeCode(object request)

```

Comment:
We can move this to a separate file but, according to the [rule of three](http://c2.com/cgi/wiki?RuleOfThree), we can wait.